tests/subsys/audio_module/src/audio_module_test_common.c

```diff
@@ -8,7 +8,8 @@
 #include <zephyr/fff.h>
 
 #include "audio_module_test_common.h"
-#include "audio_module/audio_module.h"
+#include "audio_module.h"
+#include "fakes.h"
```

Comment:
"Fakes" is quite generic. Should this have a more specific name?

Reply:
Renamed audio_module_test_fakes.[c,h]